languages/elixir/exercises/concept/log-level/.docs/instructions.md

```diff
@@ -1,41 +1,39 @@
-In this exercise you'll be mapping logging codes to a label.
-
-These are the different log codes:
-
-- 0 (trace)
-- 1 (debug)
-- 4 (info)
-- 5 (warning)
-- 6 (error)
-- 7 (fatal)
-
-You have three tasks:
+You are running a system that consist of a few applications producing many logs. You want to write a small program that will aggregate those logs and give them labels according to their severity level. All applications in your system use the same log codes, but some of the legacy applications don't support all the codes.
+
+| Log code | Log label | Supported in legacy apps? |
+| -------- | --------- | ------------------------- |
+| 0        | trace     | no                        |
+| 1        | debug     | yes                       |
+| 2        | info      | yes                       |
+| 3        | warning   | yes                       |
+| 4        | error     | yes                       |
+| 5        | fatal     | no                        |
+| ?        | unknown   | -                         |
```

Comment:
I couldn't find any source for the peculiar values 0, 1, 4, 5, 6, 7. I only found the syslog standard that uses values 0-7, but in the opposite order (0 - emergency, 7 - debug). So I allowed myself to modify the numbers to remove the gap.